scene/gui/text_edit.cpp

```diff
@@ -7959,7 +7961,8 @@ void TextEdit::_update_wrap_at_column(bool p_force) {
 			text.set_width(-1);
 		}
 
-		text.invalidate_all_lines();
+		if (prev_wrap_width != text.get_width() || prev_brk_flags != text.get_brk_flags() || line_wrapping_mode)
+			text.invalidate_all_lines();
```

Comment:
`if` blocks should always use braces in the Godot codebase - though I think this part will be removed anyway if you apply kitbdev's suggestion.
```suggestion
		if (prev_wrap_width != text.get_width() || prev_brk_flags != text.get_brk_flags() || line_wrapping_mode) {
			text.invalidate_all_lines();
		}
```

Reply:
Huh, weird that clang-format didn't catch that. 🤔 Is clang-format's `InsertBraces` configured as false to keep old code from getting dirtied?

Reply:
I think `InsertBraces` just didn't exist last time I looked, and I was quite frustrated clang-format couldn't help us enforce that rule.

I'll give it a try, it now seems to be part of the template after syncing with the clang-format 18 config in 72c183ce8ba.